### PR TITLE
Homepage and routing

### PR DESCRIPTION
**Pull request by anurag-kkumarr**
Added Homepage
Added Navigation
Added routing
Added Footer
Screenshot (55)-copy

### CODE REVIEW

File: src/component/footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.footer {
+    width: 100%;
+    background-color: #000;
```
Comment (Balakumar3018) [inline src/component/footer/Footer.css:3]: Use variable here

File: src/component/navigation/NavBar.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    padding: var(--size-zero-five);
+    margin-bottom: var(--size-zero-five);
+    background-color: white;
+    z-index: 9999;
+  }
+  
+  .app-head {
+    margin-left: var(--size-one);
+    font-size: var(--size-two);
+    color: rgba(235, 13, 13, 0.829);
```
Comment (Balakumar3018) [inline src/component/navigation/NavBar.css:17]: Use variable here

**Review by Balakumar3018 — APPROVED** [review]
Left few comments and other than that good to merge

File: src/component/navigation/NavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+          <input type="search" className="search-ip" placeholder="search.." />
+          <button className="search-btn" type="submit">
+            Search
+          </button>
+        </form>
+
+        <nav className="nav-right">
+          <a href="/" className="nav-btn">Cart</a>
+          <a href="/" className="nav-btn">wishlist</a>
+          <a href="/" className="nav-btn">login</a>
```
Comment (sumitch29) [inline src/component/navigation/NavBar.jsx]: use Routing instead of <a> use Link

**Review by sumitch29 — APPROVED** [review]
Routing is not implemented in navbar, otherwise everything is good.

**Pull request merged**